newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationPage.js

```diff
@@ -236,19 +365,25 @@ const PrivateAssetPackInformationPage = ({
   };
 
   const mediaItems = assetPack
-    ? assetPack.previewImageUrls
-        .map(url => ({
+    ? [
+        {
           kind: 'image',
-          url,
-        }))
-        .concat(
-          assetPack.previewSoundUrls
-            ? assetPack.previewSoundUrls.map(url => ({
-                kind: 'audio',
-                url,
-              }))
-            : []
-        )
+          url: privateAssetPackListingData.thumbnailUrls[0],
```

Comment:
That's a good idea I think to add the thumbnail as the first preview image

Reply:
Yes, I felt sometimes lost when clicking on a pack, because I was seeing a completely different image and thought I misclicked. + it's really well polished, so might as well use it